modules/hitl/src/backend/db.ts

```diff
@@ -42,6 +42,19 @@ export default class HitlDb {
           table.timestamp('ts')
         })
       })
+      .then(() =>
+        this.knex('hitl_messages')
+          .columnInfo('text')
+          .then(info => {
+            if (info.maxLength === null || this.knex.isLite) {
+              return
+            }
+
+            return this.knex.schema.alterTable('hitl_messages', table => {
+              table.text('text', 'longtext').alter()
```

Comment:
Are you sure longtext exist in postgres ?  https://www.postgresql.org/docs/9.6/datatype-character.html

And if knex handles it..  it'll probably convert longtext to text so this migration becomes useless ..

Reply:
tested and working